vm/src/types/slot.rs

```diff
@@ -880,13 +932,8 @@ pub trait AsSequence: PyPayload {
 
 #[pyimpl]
 pub trait AsNumber: PyPayload {
-    const AS_NUMBER: PyNumberMethods;
-
-    #[inline]
     #[pyslot]
-    fn as_number(_zelf: &PyObject, _vm: &VirtualMachine) -> &'static PyNumberMethods {
-        &Self::AS_NUMBER
-    }
```

Comment:
is this need to be changed?

Reply:
we need static variable, const variable is not globally static, it creates instance wherever is used.